.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```

Comment:
This is specific to your own environment and shouldn't be checked into the repo. Feel free to add it to your own global .gitignore though: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c